src/app/core/components/left-header/left-header.component.html

```diff
@@ -1,8 +1,7 @@
 <div class="nav-header" *ngIf="!compactMode">
   <div class="nav-control">
     <span class="nav-collapse" (click)="hideLeftMenuAndHeaders()" tabindex="0">
-      <!-- <i class="fa fa-bars"></i> -->
-      <img src="assets/images/menu-open.svg" style="width: 2rem; height: 2rem;" />
+      <i class="fas fa-expand-arrows-alt" style="font-size: 2rem;"></i>
```

Comment:
As the it extends the right part, probably it would be better if it would be on the right, and not in the part which disappears. (can be done later)

Reply:
TODO : Move the icon to the right of the title "Algorea Platform", make it a left/right chevron.

Reply:
Changed icon (to `<<`) and position of the button.

![image](https://user-images.githubusercontent.com/11016337/126195654-0ddb9dbc-239b-447f-b7b9-7664896ce8c8.png)

![image](https://user-images.githubusercontent.com/11016337/126195694-c6158f1b-288f-4a4d-be78-58607ba9fc85.png)

Other choices considered :

`fa-chevron-left` : I prefer the other one when comparing

![image](https://user-images.githubusercontent.com/11016337/126195811-a787ade8-5077-4b7b-8cd3-e21d992eda8b.png)

`fa-caret-square-left` : looks too heavy to me

![image](https://user-images.githubusercontent.com/11016337/126195898-69468886-1f71-49e0-aff6-fe23dde78a1f.png)